Content.Server/Teleportation/PortalSystem.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Content.Server.Mind.Components;
+using Content.Shared.Administration.Logs;
+using Content.Shared.Database;
+using Content.Shared.Teleportation.Systems;
+using Robust.Shared.Map;
+
+namespace Content.Server.Teleportation;
+
+public sealed class PortalSystem : SharedPortalSystem
+{
+    [Dependency] private readonly ISharedAdminLogManager _adminLogger = default!;
+
+    protected override void LogTeleport(EntityUid portal, EntityUid subject, EntityCoordinates source,
+        EntityCoordinates target)
+    {
+        if (HasComp<MindComponent>(subject))
+            _adminLogger.Add(LogType.Teleport, LogImpact.Low, $"{ToPrettyString(subject):player} teleported via {ToPrettyString(portal)} from {source} to {target}");
```

Comment:
Isn't there a shared interface for this

Reply:
put a TODO on this that it should get moved to shared at some point